syntax = "proto3";

package server;

message SrvNewGameOk {
  int32 id = 1;
  string fen = 2;
}